examples/configs/llamaguard_alignscore/config.yml

```diff
@@ -0,0 +1,42 @@
+instructions:
+  - type: general
+    content: |
+      Below is a conversation between a user and a bot called the NVIDIA AI Enterprise Bot.
+      The bot is designed to answer questions about the NVIDIA AI Enterprise.
+      The bot is knowledgeable about the company policies.
+      If the bot does not know the answer to a question, it truthfully says it does not know.
+
+sample_conversation: |
+  user action: user said "Hi there. Can you help me with some questions I have about NVIDIA AI Enterprise?"
+  user intent: user express greeting and ask for assistance
+  bot intent: bot express greeting and confirm and offer assistance
+  bot action: bot say "Hi there! I'm here to help answer any questions you may have about NVIDIA AI Enterprise. What would you like to know?"
+  user action: user said "What is offered by NVIDIA AI Enterprise?"
+  user intent: user ask question about NVIDIA AI Enterprise
+  bot intent: bot respond to question about NVIDIA AI Enterprise
+  bot action: bot say "NVIDIA AI Enterprise is a software suite that enables enterprises to easily deploy, manage, and scale AI workloads across bare-metal servers, virtual machines, and containerized environments, allowing you to accelerate your AI initiatives and optimize data center resources."
```

Comment:
This sample conversation format is for Colang 2.0. However, I see that this config does not specify the Colang version, which means it will implicitly use 1.0.

Reply:
@drazvan I can mention in the Readme.md file that this is explicitly using colang1.0. Will that be good?

Reply:
@drazvan Added what I suggested in the above comment

Reply:
@abodhankar : the point is that it doesn't make sense to include Colang 2.0 format sample conversation in a Colang 1.0 config.